### PR TITLE
LINK-71 | Event page navigation

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Component to navigate between event form pages

## Closes
**[LINK-71](https://helsinkisolutionoffice.atlassian.net/browse/LINK-71)**

## Screenshot
Näyttökuva 2020-10-19 kello 12 49 05

### CODE REVIEW

File: src/domain/event/eventNavigation/EventNavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  return (
+    <div
+      className={classNames(styles.eventNavigation, css(theme.eventNavigation))}
+    >
+      <div className={styles.eventNavigationPanel}>
+        <div className={styles.circlesWrapper} role="navigation">
+          {items.map((item, index) => {
+            return (
+              <div
+                key={index}
```
Comment (Sartonon) [inline src/domain/event/eventNavigation/EventNavigation.tsx:54]: should this handle enter key?

File: src/domain/event/eventNavigation/EventNavigation.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    if (item.disabled) return;
+
+    setActiveTab(index);
+  };
+
+  const isPreviousDisabled = !activeTab;
+  const isNextDisabled: boolean =
+    activeTab === items.length - 1 || !!items[activeTab + 1]?.disabled;
+
+  const previous = () => {
```
Comment (Sartonon) [inline src/domain/event/eventNavigation/EventNavigation.tsx:37]: handlePreviousClick might be more fitting name for this

File: src/domain/event/eventNavigation/EventNavigation.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    setActiveTab(activeTab - 1);
+  };
+
+  const next = () => {
```
Comment (Sartonon) [inline src/domain/event/eventNavigation/EventNavigation.tsx:41]: handleNextClick might be more fitting name for this

File: src/domain/event/eventNavigation/EventNavigation.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/event/eventNavigation/EventNavigation.tsx:54]: Change component to button

File: src/domain/event/eventNavigation/EventNavigation.tsx (hunk 2 above)
Comment (jorilindell) [reply src/domain/event/eventNavigation/EventNavigation.tsx]: Fixed

File: src/domain/event/eventNavigation/EventNavigation.tsx (hunk 3 above)
Comment (jorilindell) [reply src/domain/event/eventNavigation/EventNavigation.tsx]: Fixed

**Review by Sartonon — APPROVED** [review]
Good!

**Pull request merged**